cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -143,7 +143,7 @@ public static Secret buildSecret(Reconciliation reconciliation, ClusterCa cluste
             reasons.add("certificate doesn't exist yet");
             shouldBeRegenerated = true;
         } else {
-            if (clusterCa.keyCreated() || clusterCa.certRenewed() || (isMaintenanceTimeWindowsSatisfied && clusterCa.isExpiring(secret, keyCertName + ".crt"))) {
+            if (clusterCa.keyCreated() || clusterCa.certRenewed() || isMaintenanceTimeWindowsSatisfied && clusterCa.isExpiring(secret, keyCertName + ".crt")) {
```

Comment:
I wonder if changes like this really improve the readability of the code. What do other @strimzi/maintainers think? (there seem to be many of this in this PR)

Reply:
Ye I agree it might be confusing for someone. Can we suppress this check?

Reply:
Yeah.